lcls_naming_tool/tests/test_cli.py

```diff
@@ -0,0 +1,52 @@
+# Unit tests for the LCLS Naming Tool script
+
+import pytest
+import json
+import os
+from pathlib import Path
+from lcls_naming_tool.lcls_naming_tool import display_version, load_taxons, functional_component_is_valid, fungible_is_valid, constituent_component_is_valid, increment_is_valid, starts_alphanumeric, validate
+
+
+def test_display_version(capsys):
+    display_version()
+    captured = capsys.readouterr()
+    curr_ver = os.popen('git describe --tags').read().strip().split('-')
+    assert str(captured.out).strip() == curr_ver[0]
+
+
+load_taxons()
```

Comment:
Small optimization for later: there's probably a better time and place to load the taxons. Perhaps this should be done automatically when checking anything if it hasn't been done yet.